source/reference/ulimit.txt

```diff
@@ -140,15 +146,57 @@ For many distributions of Linux you can change values by substituting
 the ``-n`` option for any possible value in the output of ``ulimit
 -a``.
 
-After changing the ``ulimit`` settings, you *must* restart the
-process to take advantage of the modified settings. On Linux, you can
-use the ``/proc`` file system to see the current limitations on a
-running process.
+When ``ulimit`` is used to change resource limits in a shell, the changes
+affect only the processes subsequently launched by that shell process.
+This means:
+
+- If you launch :binary:`~bin.mongod` or :binary:`~bin.mongos` manually from
+  a shell, you must start new :binary:`~bin.mongod` or :binary:`~bin.mongos`
+  processes after changing resource limits via ``ulimit``.
+
+- If :binary:`~bin.mongod` or :binary:`~bin.mongos` are launched using
+  startup scripts, the ``ulimit`` invocation must generally be placed in those
+  startup scripts prior to :binary:`~bin.mongod` or :binary:`~bin.mongos`
+  invocations.
```

Comment:
Unless users  installed MDB via the TGZ, and are creating their own init scripts (init.d, systemd, upstart), the appropriate ulimit values are already present in init scripts provided with the RPM or DEB file they installed. 

It's still valuable to discuss what to do for users with custom scripts (For example, the numactl tabs here https://docs.mongodb.com/manual/administration/production-notes/#configuring-numa-on-linux), but it is the least likely case for our users. I don't think we need anything more than a small mention. So the three options are:

1. Run from command line: you must set ulimit via the methods provided on this page.
2. Run from our init script: no action
3. Installed via TGZ and made your own init script: you must edit your init script to set ulimit